privaterelay/views.py

```diff
@@ -138,9 +139,9 @@ def invitation(request):
 def waitlist(request):
     if not settings.WAITLIST_OPEN:
         raise PermissionDenied
-    json_body = json.loads(request.body)
-    email = json_body['email']
-    fxa_uid = json_body['fxa_uid']
+    request_data = get_post_data_from_request(request)
+    email = request_data['email']
+    fxa_uid = request_data['fxa_uid']
     if not email or not fxa_uid:
         return JsonResponse({}, status=400)
```

Comment:
The changes fix joining the wait-list, but this makes the browser window show the JSON response. Should we detect a request that is not `application/json` and redirect them ... somewhere? Or just return a nicer response message?

<img width="632" alt="image" src="https://user-images.githubusercontent.com/71928/85773438-74b3e700-b6e3-11ea-8d1e-82e73b9f7abb.png">

Reply:
Good point, I think either a successful message or redirect to the home page will do but I am leaning towards nicer response message since the user may not know whether they were added to the waitlist if we just redirect the user.

Reply:
Agree just a nicer message is an improvement over what happens now (i.e., completely broken!)